app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
The `skills` attribute is defined as a class variable. This means it is shared across all instances of `SoftwareEngineer` and its subclasses. If you modify `skills` for one instance, it will affect all instances. Consider defining `skills` as an instance variable within the `__init__` method to ensure each instance has its own separate list of skills.